src/components/Flash/Flash.tsx

```diff
@@ -59,18 +60,38 @@ class FlashComponent extends React.PureComponent<MergedProps, State> {
 
   public render() {
     const { visible } = this.state
-    const { type, text, className = '', onClose, theme } = this.props
-    const iconName = type === 'success' ? 'fa-check-circle' : 'fa-exclamation-triangle'
+    const { type, text, onClose, theme, className } = this.props
+
+    let iconName: keyof typeof iconMap = 'fa-check-circle'
+    let iconColor = theme.palette.TEXT_GREY
+
+    switch (type) {
+      case 'success':
+        iconName = 'fa-check-circle'
+        iconColor = theme.palette.MAIN
+        break
+      case 'info':
+        iconName = 'fa-info-circle'
+        iconColor = theme.palette.TEXT_GREY
+        break
+      case 'warning':
+        iconName = 'fa-exclamation-triangle'
+        iconColor = theme.palette.WARNING
+        break
+      case 'error':
+        iconName = 'fa-exclamation-circle'
+        iconColor = theme.palette.DANGER
+    }
 
     if (!visible) return null
 
     return (
       <Wrapper className={`${type} ${className}`} theme={theme}>
-        <Icon name={iconName} size={24} color="#fff" />
+        <Icon name={iconName} size={14} color={iconColor} />
         <Txt theme={theme}>{text}</Txt>
-        <CloseButton onClick={onClose} className="close">
-          <Icon name="fa-times" size={12} color={theme.palette.BORDER} />
-        </CloseButton>
+        <SecondaryButton className="flash_onclose" onClick={onClose} size="s" square>
```

Comment:
Was it inconvenient if the class name was `close` ?
I think you don't have to change from `close`.

Reply:
Or you can use styled component like that.
```
const CloseButton = styled(SecondaryButton)`
   position: absolute;
   top: 50%;
   ...
`
```

Reply:
> Or you can use styled component like that.

ahhh.. it's great idea!